ask-sdk-webservice-support/ask_sdk_webservice_support/verifier.py

```diff
@@ -351,7 +351,7 @@ def _validate_cert_chain(self, cert_chain):
 
             validator = CertificateValidator(end_cert, intermediate_certs)
             validator.validate_usage(key_usage={'digital_signature'})
-        except PathValidationError as e:
+        except (PathError, ValidationError) as e:
```

Comment:
So if either PathError or ValidationError is raised by call the same VerificationException message?

Reply:
Well, at the end of the day, we want the same `VerificationException` from the `ask-sdk-webservice-support` package to be raised to the end user. Since we are passing the actual exception `e` to the error, the error message that lead to this exception will be communicated with the end user. Since we want a seamless experience for our end users to upgrade to this package, I thought it is better to not add new exceptions and instead use the same one + pass the right info. Do you think we need to handle this differently?

Reply:
Hmm I see, since this is an internal method call too, raising same exception seems ok with actual exception being logged. As long as we are just raising exceptions and not having any code changes/flow following that, this change seems ok. 
Thanks for the explanation.